atomix/cluster/src/main/java/io/atomix/raft/protocol/PersistedRaftRecord.java

```diff
@@ -1,19 +1,11 @@
 /*
- * Copyright © 2020 camunda services GmbH (info@camunda.com)
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
```

Comment:
:x: Why this change? Since when we have here a different License?

Reply:
Ah, it seems I IntelliJ added it automatically. I thought it only did that on new files, not when moving files...anyway it wasn't on purpose at all, sorry

Reply:
I guess we need to fix this, otherwise, we have the project everywhere different license headers in the modules.

Reply:
I'll look at how this happened. I'm 100% sure I didn't create a new file or copy this manually...I'll try to reproduce it :thinking: